Library/ViewModels/ThanksViewModel.swift

```diff
@@ -178,18 +178,27 @@ public final class ThanksViewModel: ThanksViewModelType, ThanksViewModelInputs,
         )
       }
 
-    project
-      .takeWhen(self.goToProject)
-      .observeValues { project in
-        AppEnvironment.current.koala.trackCheckoutFinishJumpToProject(project: project)
-      }
-
     project
       .takeWhen(self.showRatingAlert)
       .observeValues { project in
         AppEnvironment.current.koala.trackTriggeredAppStoreRatingDialog(project: project)
       }
 
+    self.projectTappedProperty.signal.skipNil().map { project in
+      let recommendedParams = DiscoveryParams.defaults
```

Comment:
Is this to match it to `recommendedParams` below on line 296?

Perhaps we should extract that to a global variable to be sure they are they same?